charts/postgres_config/templates/job.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.job.name }}
+  namespace: {{ .Values.psql_namespace }}
+spec:
+  template:
+    spec:
+      containers:
+      - name: db-setup
+        image: schulcloud/infra-tools:{{ .Values.psql.imageTag }}
+        command: ["/bin/bash", "-c", "/scripts/config-script"]
+        resources: {{ toYaml .Values.psql.resources | nindent 12 }}
+        volumeMounts:
+        - name: config-script-volume
+          mountPath: /scripts
+        envFrom:
+        - secretRef:
+            name: psql-config-secret  
+        env:
+        - name: DB_NAME
+          value: {{ .Values.database.name }}  
+      volumes:
+      - name: config-script-volume
+        configMap:
+          name: psql-config-script
```

Comment:
Where does this configmap come from?

Reply:
They come from infra-dbp
https://github.com/dBildungsplattform/infra-dbp/tree/dbp-554-Automate-Database-Creation-for-Each-Dev-Branch-in-Pg-Cluster/ansible/group_vars/spsh

Reply:
Why isn't it part of the Helm Chart?

Reply:
it may be better to place the configmap into the Helm Chart, there is no critical information is it? ,a script i think should not be placed in a group var

Reply:
we only need them once( ConfigMap, secret). Therfore we generate them during  cluster setup with infra-dbp.In the spsh-app-deploy pipeline, we  install only the Helm chart, assuming the ConfigMap and secret are already in the cluster.

Reply:
I think this makes the Helm chart less easy to understand, it's fine if we have the same configmap multiple times, if it means the code is more clear. The Helm chart should also have an existingSecret variable with a comment in the values.yml that explains the expected format content of the secret.

Reply:
@aimee-889, Regarding the script, I initially placed it in group_var, following the example of the keycloak_backup_script found in infra-schoolcloud. If putting the script in group_var is not suitable, where would you recommend it be located?

Reply:
I will add a secret and a config-map template in the helm-chart and I will use the existingSecret variable for the secret. The Secret should in my opinion nevertheless be installed in the cluster via infra-dbp, together with the postgresql-cluster creation, so everything for a recreation of the spsh-dev-cluster is in the Code.